doc/source/getting_started/faqs.rst

```diff
@@ -0,0 +1,235 @@
+.. _faqs:
+
+Frequently asked questions
+==========================
+
+What is PyAnsys?
+----------------
+PyAnsys is a set of open source technologies that allow you to interface Pythonically
+with Ansys Fluent, Mechanical APDL, AEDT, and other Ansys products. You can use PyAnsys
+libraries within a Python environment of your choice in conjunction with external Python
+libraries.
+
+.. image:: ../_static/PyAnsys_overview.png
+  :width: 800
+  :alt: PyAnsys overview
+
+What is PyFluent?
+-----------------
+PyFluent provides Pythonic access to Ansys Fluent. Its features enable the seamless use of
+Fluent within the Python ecosystem and broad access to native Fluent features for performing
+actions such as these:
+
+- Launch Fluent using a local Ansys installation.
+- Use Fluent's TUI (text user interface) commands for both meshing and solver features.
+- Use Fluent's built-in postprocessing capabilities.
+
+PyFluent is not bundled with the Fluent installation. You must download and install PyFluent
```

Comment:
@PipKat @dnwillia-work this statement is also out of date now.

Reply:
Yeah, where is it sitting in the installation?

Reply:
@seanpearsonuk @dnwillia-work I noted that it was included--but not the where. I can add this information if someone provides it.